app/swagger/requests/ppiu.rb

```diff
@@ -22,6 +22,37 @@ class PPIU
             end
           end
         end
+
+        operation :post do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Update a veterans payment information'
+          key :operationId, 'postPaymentInformation'
+          key :tags, %w[form_526]
+
+          parameter :authorization
+
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Payment information to be updated for the user'
+            key :required, true
+
+            schema do
+              property :account_type, type: :string, example: 'Checking'
+              property :financial_institution_name, type: :string, example: 'Bank Of America'
+              property :account_number, type: :string, example: '1234567890'
+              property :financial_institution_routing_number, type: :string, pattern: /^\d{9}$/, example: '123456789'
```

Comment:
Nice job on including the `pattern` in here!

Perhaps add the pattern for `account_number`, as we're validating its format, too?